common/test/acceptance/pages/lms/learner_profile.py

```diff
@@ -163,7 +163,7 @@ def field_is_visible(self, field_id):
         Returns:
             True/False
         """
-        self.wait_for_ajax()
+        self.wait_for_field(field_id)
```

Comment:
Scout rule: waiting on specific things is always preferable to a bare `wait_for_ajax()`